maisim/maisim.Game/Store/MaisimFilePath.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.IO;
+
+namespace maisim.Game.Store
+{
+    public class MaisimFilePath
+    {
+        public static string BeatmapFilePath = Path.Combine(
```

Comment:
This assumes the beatmap database is located at the default location, but this may not hold true in the future, if we allow migrating the data files to another place.

Reply:
Added `BeatmapDatabaseContextFactory` that will contain all database method here. 👍

Reply:
As Discord discussion, described in https://github.com/HelloYeew/maisim/issues/141